runtime/oti/jclprots.h

```diff
@@ -309,6 +309,13 @@ extern J9_CFUNC jint
 J9SigQuitStartup (J9JavaVM * vm);
 #endif /* J9VM_INTERP_SIG_QUIT_THREAD (autogen) */
 
+/* J9SourceSigUsr2 */
```

Comment:
If this is a reference to the file where these functions are defined, it should say `sigusr2.c`.